test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -554,7 +554,8 @@ function (Q, OktaAuth, WidgetUtil, Okta, Util, AuthContainer, IDPDiscoveryForm,
     });
 
     Expect.describe('Device Fingerprint', function () {
-      itp('contains fingerprint header in get security image request if feature is enabled', function () {
+      itp(`contains fingerprint header in get security image request if both features(
```

Comment:
nit: can we duplicate this test so we can have
- explicitly set `useDeviceFingerprintForSecurityImage=true`
- check that the default of `useDeviceFingerprintForSecurityImage` is true

so basically
- "contains fingerprint header in get security image request if both features are enabled"
- "contains fingerprint header in get security image request if deviceFingerprinting is true (deviceFingerprinting defaults to true)`